source/gui/settingsDialogs.py

```diff
@@ -3295,6 +3295,23 @@ def __init__(self, parent):
 
 		self.Layout()
 
+		silenceDurationLabelText = _(
+			# Translators: The label for a setting in advanced panel to change silence duration
+			"Duration in seconds of silence or white noise played to keep audio device open (or 0 to disable)"

```

Comment:
1. I would totally drop "white noise" from the GUI. I'd recommend to add it in the developer's guide with other hidden config parameters (maybe in a separate PR though).
2. It's quite obvious that setting to 0 will add no silence at all. You can add "0 to disable" in the User Guide, but IMO, the GUI should not be cluttered with this detail.

```suggestion
			"Duration in seconds of silence to keep audio device open"
```